SmartDeviceLink/SDLStreamingMediaConfiguration.h

```diff
@@ -85,6 +85,11 @@ typedef NS_ENUM(NSUInteger, SDLCarWindowRenderingType) {
  */
 @property (assign, nonatomic) BOOL allowMultipleViewControllerOrientations;
 
+/**
+ When YES, the StreamingMediaManager will send a black screen with "Video Backgrounded String". Defaults to YES.
+ */
+@property (assign, nonatomic) BOOL showVideoBackgroundDisplay;
```

Comment:
@t-yoshii After more thought and using this, I think this approach won't work. It does not give apps the ability to modify if they show the background display after connection time, which is necessary to support requirements for all OEMs (some OEMs require the display, and others require that it not be there). I think we'll need to have a proposal revision to change this. Let me know if you'd like me to do it or if you'd like to do it yourself.

Reply:
Oh, you mean that the `showVideoBackgroundDisplay` is set at the initialization of lifecycle managers, however, HU type is known after RAI (later than initialization of lifecycle managers.)?
That's a good point.

I would like to hear your fix plan.

Reply:
I have submitted a proposal revision https://github.com/smartdevicelink/sdl_evolution/pull/796

Reply:
@joeljfischer 
I read your proposal and agree with the change. Thank you.

Reply:
Hello @t-yoshii, the proposal revision has been accepted! Would you like to make the changes in this PR, or submit a new PR, or I can submit a PR when I have time.

Reply:
@joeljfischer ,
I updated the PR to reflect the revision. Could you please review?

Thanks